blocks/front-page/hero/hero.module.css

```diff
@@ -0,0 +1,88 @@
+@import "@jetbrains/kotlin-web-site-ui/out/components/breakpoints/media.pcss";
+
+.heroSection {
+    background: #19191C;
+    padding: 96px 0;
+    overflow: hidden;
+
+    @media (--ktl-tm) {
```

Comment:
Just a proposal.
I prefer to group all the media rules below rather to add them in every CSS rule set.
This helps you see together all the changes that fall on a given screen size.